sparse/sfold.go

```diff
@@ -63,25 +63,33 @@ func coalesce(parentFileIo FileIoProcessor, childFileIo FileIoProcessor) error {
 		}
 
 		// now we have a data start offset and length(hole - data)
-		// let's read from child and write to parent file block by block
+		// let's read from child and write to parent file. We read/write up to
```

Comment:
It's not related to ssync change right? Please put sfold change in an separate commit.